shared/teams/team/members-tab/member-row/index.tsx

```diff
@@ -103,19 +103,19 @@ export const TeamMemberRow = (props: Props) => {
                 ]}
               />
             </Box>
-            <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+            <Box style={{...Styles.globalStyles.flexBoxRow, alignItems: 'center'}}>
               {fullNameLabel}
               {crown}
               {!active && (
                 <Text
                   type="BodySmall"
                   style={{
-                    ...globalStyles.fontBold,
-                    backgroundColor: globalColors.red,
-                    color: globalColors.white,
-                    marginRight: globalMargins.xtiny,
-                    paddingLeft: globalMargins.xtiny,
-                    paddingRight: globalMargins.xtiny,
+                    ...Styles.globalStyles.fontBold,
```

Comment:
I think we should move these objects into `styles`

Reply:
moved this one but just realized there are more; please hold off review for a sec;

Reply:
ok ready again!